DllImportGenerator/TestAssets/NativeExports/NativeExports.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="DNNE" Version="1.*" />
+  </ItemGroup>
+
+  <!--
+    Consuming this project's generated native exports requires the
+    native binary and the generated *.runtimeconfig.json file.
+  -->
+  <ItemGroup>
+    <None Include="$(OutputPath)$(DnneNativeExportsBinaryName)$(DnneNativeBinaryExt)">
+      <Link>$(DnneNativeExportsBinaryName)$(DnneNativeBinaryExt)</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </None>
+
+    <!--
+      The order of copy for None items and generation of the runtimeconfig.json files
+      are inverted and I am unable to figure out the best way to circumvent this issue.
+      As a mitigation I am checking if the file exists and only adding it in that case.
+      This requires building twice from a clean environment, but with this hack a
+      ProjectReference to this project works as desired.
+    -->
+    <None Include="$(OutputPath)$(TargetName).runtimeconfig.json"
+        Condition="Exists('$(OutputPath)$(TargetName).runtimeconfig.json')">
```

Comment:
Any suggestions are welcome to avoid this issue. It works perfectly after a second build, but is obnoxious.

Reply:
Maybe something like this: https://github.com/dotnet/runtime/blob/fef560c36610493a0b09542f1e70f74386dc57be/eng/testing/runtimeConfiguration.targets#L16-L30

Reply:
That specific approach didn't work, but there was another option in the referenced issue that did. Thanks. Now it just works like any other project reference - w00t!